caps/views.py

```diff
@@ -368,7 +373,21 @@ def get_context_data(self, **kwargs):
                 for council in councils
                 if council.combined_authority
             ]
-            context["councils"] = list(councils) + combined_authorities
+            councils = list(councils) + combined_authorities
+
+            # Fetch successors for any replaced councils, even if mapit doesn't know about them yet
+            for c in councils:
+                if c.replaced_by:
+                    councils.extend(list(c.get_successors()))
+            # remove any councils that are former councils
+            councils = [
+                x
+                for x in councils
+                if x.id in Council.current_councils().values_list("id", flat=True)
+            ]
+            # remove duplicates based on x.id
+            councils = list({x.id: x for x in councils}.values())
```

Comment:
if we made councils a set then we could skip this step.

Reply:
I think this case, making a set of django models ends up being unique on memory instances rather than the ID.  Could be wrong about that.